src/main/java/frc/robot/maps/OffAxis.java

```diff
@@ -32,7 +33,7 @@ public SwerveDriveMap getDriveMap() {
         // Value taken from CAD as offset from center of module base pulley to center
         // of the robot
         final double MODULE_OFFSET_XY = 0.381;
-        final PigeonGyro pigeonGyro = new PigeonGyro(new PigeonIMU(5));
```

Comment:
This should be changed back to the original. OffAxis has a PigeonGyro not a PigeonGyro2.